backend_jobs/verify_labels/tests/redefine_labels_tests.py

```diff
@@ -0,0 +1,71 @@
+"""
+  Copyright 2020 Google LLC
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+"""
+
+import unittest
+from backend_jobs.verify_labels.pipeline_lib.redefine_labels import RedefineLabels, get_redefine_map
+
+class TestRedefineLables(unittest.TestCase):
+    """ Tests FilterBy interface implementations.
+
+    """
+
+    def setUp(self):
+        self.cat_image = {'url':'https://live.staticflickr.com/3677/13545844805_170ec3746b_c.jpg',\
+            'imageAttributes':{'format': 'jpg', 'resolution': {'width': 800, 'height': 600}},\
+                'ingestedProviders': [], 'labelName': 'cat'}
+        self.cats_image = {'url':'https://live.staticflickr.com/3677/13545844805_170ec3746b_c.jpg',\
+            'imageAttributes':{'format': 'jpg', 'resolution': {'width': 800, 'height': 600}},\
+                'ingestedProviders': [], 'labelName': 'cats'}
+        self.dog_image =  {'url':'https://live.staticflickr.com/5284/5338762379_59f7435b93_c.jpg',\
+            'imageAttributes': {'format': 'pdf', 'resolution': {'width': 800, 'height': 600}},\
+                'ingestedProviders': [], 'labelName': 'dog'}
+        provider_id = 'For_Tests' # A document in the Firestore database designated for tests.
+        self.redefine_map = get_redefine_map(provider_id)
```

Comment:
this should probably also be mocked, so we don't have to rely on a certain database state, but instead hardcode the mapping in the tests.

Reply:
On another look I figured out the it is possible to just create the map here and check the redefining only - so no need of mock and no need of firebase.